### PR TITLE
Add Product Code: Sejda.PDFDesktop version 6.0.8.0

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/23016)

### CODE REVIEW

File: manifests/s/Sejda/PDFDesktop/6.0.8.0/Sejda.PDFDesktop.yaml [hunk 1]
```diff
@@ -1,26 +1,9 @@
+# Created using YamlCreate.ps1 v1.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
 PackageIdentifier: Sejda.PDFDesktop
-Publisher: Sejda
-PackageName: PDF Desktop
 PackageVersion: 6.0.8.0
```
Comment (ItzLevvie) [inline manifests/s/Sejda/PDFDesktop/6.0.8.0/Sejda.PDFDesktop.yaml:5]:
```suggestion
PackageVersion: 6.0.8
```

File: manifests/s/Sejda/PDFDesktop/6.0.8.0/Sejda.PDFDesktop.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# Created using YamlCreate.ps1 v1.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Sejda.PDFDesktop
+PackageVersion: 6.0.8.0
```
Comment (ItzLevvie) [inline manifests/s/Sejda/PDFDesktop/6.0.8.0/Sejda.PDFDesktop.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 6.0.8
```

File: manifests/s/Sejda/PDFDesktop/6.0.8.0/Sejda.PDFDesktop.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# Created using YamlCreate.ps1 v1.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Sejda.PDFDesktop
+PackageVersion: 6.0.8.0
```
Comment (ItzLevvie) [inline manifests/s/Sejda/PDFDesktop/6.0.8.0/Sejda.PDFDesktop.installer.yaml:5]:
```suggestion
PackageVersion: 6.0.8
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
image

**Comment by vedantmgoyal2009** [comment]
Why did it not remove `Needs: author feedback` and `Needs-Author-Feedback` labels?

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**